apps/crm/src/modules/mailchimp/views/mailchimp-config-view/mailchimp-config-view.tsx

```diff
@@ -0,0 +1,75 @@
+import { trpcClient } from "../../../trpc/trpc-client";
+import { Box, Text } from "@saleor/macaw-ui/next";
+import { RemoveMailchimpConfig } from "../../config-sections/remove-mailchimp-config/remove-mailchimp-config";
+import { MailchimpAuthorizeView } from "../mailchimp-authorize-view/mailchimp-authorize-view";
+import { MailchimpLists } from "../../config-sections/mailchimp-lists/mailchimp-lists";
+import { WebhookConfiguration } from "../../config-sections/webhook-configuration/webhook-configuration";
+import { SaleorCustomersSync } from "../../../saleor-customers-sync/saleor-customers-sync";
+import { ConnectionStatus } from "../../config-sections/connection-status/connection-status";
+import { Instructions } from "../../config-sections/instructions/instructions";
+
+const header = (
+  <Box marginBottom={12}>
+    <Text as="h1" variant="title" size="large">
+      Mailchimp
+    </Text>
+  </Box>
+);
+
+export const MailchimpConfigView = () => {
+  const {
+    data: mailchimpConfigured,
+    refetch,
+    isFetched,
+    isLoading,
+    error,
+  } = trpcClient.mailchimp.config.getMailchimpConfigured.useQuery();
+
+  const isMailchimpConfigured = mailchimpConfigured?.configured;
+
+  if (isLoading) {
+    return (
+      <div>
+        {header}
+        <Text>Checking Mailchimp config status...</Text>
+      </div>
+    );
+  }
+
+  if (error) {
+    return (
+      <div>
+        {header}
+        <Box marginBottom={12} display="flex" justifyContent="space-between" gap={12}>
+          <ConnectionStatus status="error" __flex="0 1 50%" />
+        </Box>
+      </div>
+    );
+  }
+
+  if (isMailchimpConfigured && isFetched) {
+    return (
+      <div>
+        {header}
+        <Instructions marginBottom={12} />
+        <Box marginBottom={12} display="flex" justifyContent="space-between" gap={12}>
+          <ConnectionStatus status="ok" __flex="0 1 50%" />
+          <MailchimpLists __flex="0 1 50%" />
+        </Box>
+
+        <WebhookConfiguration marginBottom={12} />
+        <SaleorCustomersSync marginBottom={12} />
+        <RemoveMailchimpConfig />
```

Comment:
Similar comment here. We can control the spacing with a separate `Box` component and a `gap` property on it (like in line 55). IMO, allowing to pass the styles props to `Section` to `Box` will make it harder to maintain.

Reply:
I am all for redundancy here. IMO, the influence on the display of the `SaleorCustomerSync` should be minimal and explicit, as it is a custom component. `Box` is generic by design, you are expected to use it to provide one-time styles. 

If all our components have this level of customizability, we never truly know what they look like.

Reply:
So what API would you propose (I still disagree ;) )?